angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.component.ts

```diff
@@ -51,7 +51,7 @@ import { ISprkMastheadSelectorChoice } from '../sprk-masthead-selector/sprk-mast
           <a
             sprkLink
             variant="plain"
-            class="sprk-o-Stack sprk-o-Stack--split@xxs sprk-o-Stack--center-column sprk-u-Width-100"
+            class="sprk-o-Stack sprk-o-Stack--split@xxs sprk-o-Stack--center-column sprk-u-Width-100 sprk-c-Masthead__selector-dropdown-header-link"
```

Comment:
This is a new class used to apply styles to these links.

Reply:
The sprk-u-Width-100 is not needed here because Stack applies width 100%.